jarbas/core/migrations/0024_add_available_in_latest_dataset_field_to_reimbursement.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
My doubt is about why do not create these two new fields with only one migration?